### PR TITLE
Track: ztimer replace xtimer

### PR DESCRIPTION
**Pull request by fjmolinas**
### Contribution description

I simply ran:

```
$ spatch --sp-file sys/ztimer/xtimer2ztimer.cocci --macro-file-builtins dist/tools/coccinelle/include/riot-standard.h . -in_place       -include_headers
```

And the replaced all `USEMODULE += xtimer` by `USEMODULE += ztimer_usec`.

lets see how far away we still are...

### Testing procedure

Lets see what murdock thinks...

### CODE REVIEW

**Comment by fjmolinas** [comment]
Un-tagged everyone to avoid spamming..

File: sys/include/xtimer/implementation.h [hunk 1]
```diff
@@ -211,22 +211,24 @@ static inline void xtimer_periodic_wakeup(xtimer_ticks32_t *last_wakeup, uint32_
     _xtimer_periodic_wakeup(&last_wakeup->ticks32, _xtimer_ticks_from_usec(period));
 }
 
-static inline void xtimer_set_wakeup(xtimer_t *timer, uint32_t offset, kernel_pid_t pid)
+static inline void xtimer_set_wakeup(ztimer_t *timer, uint32_t offset,
```
Comment (miri64) [inline sys/include/xtimer/implementation.h:214]: :raised_eyebrow: ? Is that correct?

Comment (fjmolinas) [reply sys/include/xtimer/implementation.h:214]: nope, the script shouldn't hava run here but I thought I had fixed this...

**Comment by kaspar030** [comment]
> Ahh nice now it actually ran for a bit before hitting too many failures :D

hehe. for faster turn-around, may I suggest adding "BOARDS=native" to .murdock?

**Comment by fjmolinas** [comment]
Superseded by #17365

**Pull request closed without merging**